README.md

```diff
@@ -45,31 +45,38 @@ Tart Executor is open sourced under MIT license so people can base their own exe
 Tart itself on the other hand is [source available under Fair Software License](https://tart.run/licensing/)
 that required paid sponsorship upon exceeding a free limit.
 
-## Advanced configuration
+## Supported command-line arguments
 
-By default, for preparing a virtual machine and running scripts inside of it Tart Executor assumes that
-your virtual machines can be SSH-ed into `admin` user via `admin` password.
+### `prepare` stage
 
-```toml
-[runners.custom]
-  prepare_exec = "gitlab-tart-executor"
-  prepare_args = ["prepare", "--username=gitlab", "--password=password101"]
-  run_exec = "gitlab-tart-executor"
-  run_args = ["run", "--username=gitlab", "--password=password101"]
-  cleanup_exec = "gitlab-tart-executor"
-  cleanup_args = ["cleanup"]
-```
+| Argument  | Default           | Description                                                     |
+|-----------|-------------------|-----------------------------------------------------------------|
+| `--cpu`   | `0` (no override) | Override default image CPU configuration (number of CPUs)       |
+| `--memory` | `0` (no override) | Override default image memory configuration (size in megabytes) |
 
-`prepare` command has arguments to override CPU/Memory of cloned virtual machines.
-For a full list of options please refer to `gitlab-tart-executor prepare --help`.
+## Supported environment variables
```

Comment:
I think it will be nice as part of #10 to have an example where `TART_EXECUTOR_HOST_DIR` is set for a job and SSH credentials are overridden for people to copy-and-paste.

Reply:
I think it's worth adding it right in this PR which introduce the option to configure via environment variables.

Reply:
See https://github.com/cirruslabs/gitlab-tart-executor/pull/11/commits/777f49e5fa3ee6852edb7db3b190538db1349e33.